.ci/jenkins/base_image.dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:16.04
+
+ARG http_proxy
+ARG https_proxy
+ENV http_proxy=${http_proxy} https_proxy=${https_proxy}
+
+# nGraph dependencies
+RUN apt-get -y update && apt-get -y install git build-essential cmake clang-3.9 git curl zlib1g zlib1g-dev libtinfo-dev \
+                                            python python3 python-pip python3-pip python-dev python3-dev python-virtualenv \
+                                            protobuf-compiler libprotobuf-dev
+
+RUN pip install --upgrade pip setuptools wheel && pip3 install --upgrade pip setuptools wheel
+
+WORKDIR /root
```

Comment:
you can remove this workdir section and use 
```
git clone git@...repo.. /full/path/to/repo
```

Reply:
done